README.md

```diff
@@ -53,43 +59,41 @@ fud exec depth.futil --to interpreter-out -s verilog.data depth.data > depth.txt
 python3 parse_data.py -di temp.txt
 ```
 
+What just happened? Below, we walk through the six commands we issued above, pointing out the other options that we could have used.
+
 First, `make fetch` downloads some [GFA][] data files into the `./test` directory.
 
-To build the odgi graph files from the GFA files, run `make test/*.og`.
+Second, `make test/*.og` builds the odgi graph files from those GFA files.
 
-Then, `python3 calyx_depth.py -o depth.futil` generates the hardware accelerator and writes it to a file named `depth.futil`. The commands to generate a node depth hardware accelerator in calyx include:
+Third, we generate the hardware accelerator and write it to a file named `depth.futil`. The commands to generate a node depth hardware accelerator in calyx include:
 
 1. `python3 calyx_depth.py -o depth.futil`
-2. `python3 calyx_depth.py -a <filename> -o depth.futil`
-3. `python3 calxy_depth.py -n=MAX_NODES -e=MAX_STEPS -p=MAX_PATHS -o depth.futil`
+2. `python3 calxy_depth.py -n=MAX_NODES -e=MAX_STEPS -p=MAX_PATHS -o depth.futil`
+3. `python3 calyx_depth.py -a <filename.og> -o depth.futil`
 
 The commands use the hardware parameters as follows:
-1. Uses default hardware parameters
-2. Automatically infers the hardware parameters from a `.og` file
-3. Takes the hardware parameters as input.
+1. Uses default hardware parameters.
+2. Takes the hardware parameters as input.
+3. Automatically infers the hardware parameters from a `.og` file.
 
-Automatically inferred parameters take precedence over manually specified ones, and a subset of parameters may be specified. For example, `python3 calyx_depth.py -a test/k.og -n=1` will infer `MAX_STEPS` and `MAX_PATHS` from `test/k.og`, but the resulting accelerator can only handle one node.
+Automatically-inferred parameters take precedence over manually specified ones, and a subset of parameters may be specified. For example, `python3 calyx_depth.py -a test/k.og -n=1` will infer `MAX_STEPS` and `MAX_PATHS` from `test/k.og`, but the resulting accelerator can only handle one node.
 
-To run the hardware accelerator, we need to generate some input using one of the following commands:
+Fourth, we need to generate some input from our odgi file. This is what we will feed to the hardware accelerator. The following variations all accomplish this:
 
 1. `python3 parse_data.py <filename> -o depth.data`
 2. `python3 parse_data.py <filename> -a <filename2> -o depth.data`
```

Comment:
2 and 3 were swapped above. For consistency, we should have them in the same order in both places

Reply:
Thanks for the catch; done!